client/src/document/ingredients/spec-section.tsx

```diff
@@ -0,0 +1,67 @@
+import { DisplayH2, DisplayH3 } from "./utils";
+
+export function SpecificationSection({
+  id,
+  title,
+  isH3,
+  specifications,
+  query,
+}: {
+  id: string;
+  title: string;
+  isH3: boolean;
+  specifications: Array<{
+    title: string;
+    bcdSpecificationURL: string;
+    shortTitle: string;
+  }>;
+  query: string;
+}) {
+  return (
+    <>
+      {title && !isH3 && <DisplayH2 id={id} title={title} />}
+      {title && isH3 && <DisplayH3 id={id} title={title} />}
+
+      {/* XXX We could have a third condition; the specURL worked but yielded
+      exactly 0 specifications. If that's the case, perhaps the messaging
+      should be different. */}
```

Comment:
I think we should address this comment. `XXX` comments are not supposed to go into `main`. They're useful for drafts and stuff. 
What would be useful is to state, in a code comment, that we're aware that it could be an array of length 0 and that that should be treated as if the specifications can't be found at all.

Reply:
I'm happy to remove this comment (I think you added it). I'm not quite sure if there is a third condition to be described here. I mean, either we collected specifications using BCD and browser-specs, or that failed and so we show the warning. But maybe I'm not realizing what you meant by this comment exactly. Can you elaborate or should I just remove this comment?

Reply:
I removed the comment